packages/analytics-js-common/src/types/LoadOptions.ts

```diff
@@ -127,4 +144,6 @@ export type LoadOptions = {
   dataPlaneEventsBufferTimeout?: number;
   storage?: StorageOpts;
   preConsent?: PreConsentOptions;
+  // transport mechanism to be used for sending batched requests
+  transportMode?: EventsTransportMode; // Unused for now. This will deprecate the useBeacon and beaconQueueOptions
```

Comment:
Do we need to add a breaking change for this in the docs?

Reply:
This option is not implemented yet in the code. So, no changes required.